src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonStringEnumConverter.cs

```diff
@@ -54,9 +53,6 @@ public override bool CanConvert(Type typeToConvert)
         }
 
         /// <inheritdoc />
-        [DynamicDependency(
```

Comment:
Why was this removed? Is the linker recognizing these dependencies now?

Reply:
Yes - this situation is similar to the discussion in https://github.com/dotnet/runtime/pull/38216#discussion_r445670381/ Because there are `typeof` usages in the code, the linker can recognize these dependencies.